src/drafts/MarkdownViewer/_useListInteraction.ts

```diff
@@ -32,7 +32,8 @@ export const useListInteraction = ({
   // Storing the value in a ref allows not using the markdown value as a depdency of
   // onToggleItem, which would mean we'd have to re-bind the event handlers on every change
   const markdownRef = useRef(markdownValue)
-  useLayoutEffect(() => {
+
+  useEffect(() => {
     markdownRef.current = markdownValue
   }, [markdownValue])
```

Comment:
I think it's dangerous to update refs in regular effects because they can get out of date. Can we replace this with `useTrackingRef`? Pretty sure that uses `useIsomorphicLayoutEffect` under the hood

Reply:
Looks like `useTrackingRef` just uses `useLayoutEffect` not `useIsomorphicLayoutEffect`. I swapped this `useEffect` for `useIsomorphicLayoutEffect` 👍